### ISSUE TITLE
Need to Update README.MD Installation

### ISSUE THREAD

**Issue by devhyunjae** [crazycodeboy/react-native-splash-screen#4]
3.Update the MainApplication.java file to use react-native-splash-screen via the following changes:

``` java
public class MainApplication extends Application implements ReactApplication {

    private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
        @Override
        protected boolean getUseDeveloperSupport() {
            return BuildConfig.DEBUG;
        }

        @Override
        protected List<ReactPackage> getPackages() {
            return Arrays.<ReactPackage>asList(
                    new MainReactPackage(),
            new SplashScreenReactPackage()  //here
            );
        }
    };

    @Override
    public ReactNativeHost getReactNativeHost() {
        return mReactNativeHost;
    }
}
```

Need to `import` the package.

```
import com.cboy.rn.splashscreen.SplashScreenReactPackage;
```

``` java
Android:

Update the MainActivity.java file to use react-native-splash-screen via the following changes:

public class MainActivity extends ReactActivity {
   @Override
    protected void onCreate(Bundle savedInstanceState) {
        SplashScreen.show(this);  // here
        super.onCreate(savedInstanceState);
    }
    // ...other code
}
```

Also, need to import the package.

```
import com.cboy.rn.splashscreen.SplashScreen;
```

**Comment by haikyuu** [comment]
In `MainActivity.java`, you need to import other stuff too

```java
import android.os.Bundle;
import com.facebook.react.ReactActivity;
import com.cboy.rn.splashscreen.SplashScreen;
```

**Comment by crazycodeboy** [comment]
thanks，fix in #5
